runtime/jcl/module.xml

```diff
@@ -135,6 +139,14 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<makefilestubs>
 			<makefilestub data="UMA_ENABLE_ALL_WARNINGS=1"/>
 			<makefilestub data="UMA_TREAT_WARNINGS_AS_ERRORS=1"/>
+			<makefilestub data="CFLAGS+= -I/usr/include/criu">
+				<include-if condition="spec.flags.opt_criuSupport"/>
+				<include-if condition="spec.linux.*"/>
+			</makefilestub>
+			<makefilestub data="CXXFLAGS+= -I/usr/include/criu">
+				<include-if condition="spec.flags.opt_criuSupport"/>
+				<include-if condition="spec.linux.*"/>
+			</makefilestub>
```

Comment:
Rather than augment the include path (which may not work properly for cross-compile situations), why not instead of
```
#include <criu.h>
```
write
```
#include <criu/criu.h>
```